xray/backends/netcdf3.py

```diff
@@ -40,8 +40,8 @@ def coerce_nc3_dtype(arr):
         # TODO: raise a warning whenever casting the data-type instead?
         cast_arr = arr.astype(new_dtype)
         if ((('int' in dtype or 'U' in dtype) and
-                    not (cast_arr == arr).all())
-                or ('float' in dtype and
+                not (cast_arr == arr).all()) or
```

Comment:
Yes, this is recommended in [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length):
>the preferred place to break around a binary operator is after the operator, not before it.